src/NzbDrone.Core/MediaFiles/BookImport/Specifications/SameFileSpecification.cs

```diff
@@ -27,9 +27,17 @@ public Decision IsSatisfiedBy(LocalBook item, DownloadClientItem downloadClientI
 
             foreach (var bookFile in bookFiles)
             {
-                if (bookFile.Size == item.Size)
+                if (bookFile == null)
                 {
-                    _logger.Debug("'{0}' Has the same filesize as existing file", item.Path);
+                    var book = localBook.Book;
+                    _logger.Trace("Unable to get book file details from the DB. BookId: {0} BookFileId: {1}", book.Id, bookFile.Id);
```

Comment:
`bookFile` is always null here, thus remove it from the log out.

Reply:
Commited